specs/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  globals: {
+    describe: 'write',
+    it: 'write',
+    expect: 'write',
```

Comment:
I think you can add the `mocha` environment to handle all mocha globals.
```js
{
  env: {
    mocha: true
  }
}
```

Reply:
Great call!